renderer/shaders/atomic_draw.glsl

```diff
@@ -316,7 +316,7 @@ half4 resolve_path_color(half coverageCount,
     half coverage = abs(coverageCount);
 #ifdef @ENABLE_EVEN_ODD
     if ((paintData.x & PAINT_FLAG_EVEN_ODD) != 0u)
-        coverage = 1. - abs(fract(coverage * .5) * 2. + -1.);
+        coverage = 1. - make_half(abs(fract(coverage * .5) * 2. + -1.));
```

Comment:
I think both `fract` (HLSL `frac`) and `abs` will need overloads for `half`.

Reply:
For these `half` overloads of intrinsics, are you okay with using `make_half` within the function, or do you want me to write them totally from scratch?

e.g. for GLSL, `fract` is `x - floor(x)`. In HLSL, `floor` still returns a `float`, so it would need to be `x - make_half(floor(x))` unless we also want to reimplement `floor`.

Reply:
Let's reimplement overloads for floor/fract/et. al. I'd rather not pepper with extra `make_half` code for lower DX versions. It's also more future proof since we don't have these runners on the github workflows (yet).